index.html

```diff
@@ -307,6 +307,57 @@ <h2>About me</h2>
       </div>
     </main>
 
+    <footer class="about flex-column">
+      <h2 id="about__contactUs">CONTACT US</h2>
+      <p id="about__needHelp">
+        If you have an application you are interested in developing, a
```

Comment:
Kindly update the intro text of the form to better match the figma design🙏.

Your design:
<img width="236" alt="image" src="https://user-images.githubusercontent.com/37116322/218155646-ef683690-2962-4f57-b00b-55c6ae7ae6ec.png">

Figma design:
<img width="131" alt="image" src="https://user-images.githubusercontent.com/37116322/218155716-d3f0c906-ed0a-400f-ac62-f582927ab4a3.png">